ctx.go

```diff
@@ -830,16 +865,27 @@ func (ctx *Ctx) Stale() bool {
 	return !ctx.Fresh()
 }
 
+// Status sets the HTTP status for the response.
+func (ctx *Ctx) SetStatus(status int) {
+	ctx.Fasthttp.Response.SetStatusCode(status)
+}
+
 // Status sets the HTTP status for the response.
 // This method is chainable.
 func (ctx *Ctx) Status(status int) *Ctx {
-	ctx.Fasthttp.Response.SetStatusCode(status)
+	ctx.SetStatus(status)
 	return ctx
 }
 
 // Type sets the Content-Type HTTP header to the MIME type specified by the file extension.
-func (ctx *Ctx) Type(extension string) *Ctx {
+func (ctx *Ctx) SetType(extension string) {
```

Comment:
methods are based on express because our framework is designed to offer an easy changeover and the familiar handling of express
https://expressjs.com/de/4x/api.html#res.type